test/passing/open.ml.ref

```diff
@@ -167,13 +167,15 @@ let x =
 
 class type a =
   (* A'' *)
-  let open (* A' *) A (* A *) in
+  let open A (* A *) in
+  (* A' *)
   (* B *)
   b
 
 class a =
   (* A'' *)
-  let open (* A' *) A (* A *) in
```

Comment:
This is weird

Reply:
I didn't manage to fix this one, nothing seems wrong in the code.